docs/sources/articles/basics.md

```diff
@@ -35,21 +35,19 @@ and download it from [Docker Hub](https://hub.docker.com) to a local
 image cache.
 
 > **Note**:
-> When the image has successfully downloaded, you will see a 12 character
+> When the image has been successfully downloaded, you will see a 12 character
 > hash `539c0211cd76: Download complete` which is the
 > short form of the image ID. These short image IDs are the first 12
 > characters of the full image ID - which can be found using
-> `docker inspect` or `docker images --no-trunc=true`
+> `docker inspect` or `docker images --no-trunc=true`.
 
 {{ include "no-remote-sudo.md" }}
 
 ## Running an interactive shell
 
-    # Run an interactive shell in the ubuntu image,
-    # allocate a tty, attach stdin and stdout
-    # To detach the tty without exiting the shell,
-    # use the escape sequence Ctrl-p + Ctrl-q
-    # note: This will continue to exist in a stopped state once exited (see "docker ps -a")
+    # Run an interactive shell in the ubuntu image, allocate a tty, attach stdin and stdout.
```

Comment:
This really needs to be text not comments.

--> https://gist.github.com/moxiegirl/f7e77a4c04743469d3f9

To run an interactive shell in the Ubuntu image:

    $ docker run -i -t ubuntu /bin/bash		     $ docker run -i -t ubuntu /bin/bash
  
The `-i` flag starts an interactive container. The `-t` flag creates a pseudo-TTY that attaches `stdin` and `stdout`.